cost/azure/rightsize_sql_instances/azure_rightsize_sql_instances.pt

```diff
@@ -0,0 +1,273 @@
+name "Azure Rightsize SQL Instances"
+rs_pt_ver 20180301
+type "policy"
+short_description "Gathers Azure SQL database utilization data from Azure Metrics, and reports on databases that is recommended to be resized. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/azure/rightsize_sql_instances/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
use: Finds Azure SQL instances based on CPU and Memory thresholds and Resize them after approval.

Reply:
The requirement is only recommendation.  Can we agree to push the resize to a future version ?